apple-pie/styles/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+    display: flex;
+	flex-direction: column;
```

Comment:
flex-direction is not nested under display:flex, it should start from the same line